easybuild/easyconfigs/c/CellRanger-ARC/CellRanger-ARC-1.0.1.eb

```diff
@@ -0,0 +1,29 @@
+# The STAR binary included in this version has been vectorized with AVX
+# hence it is not recommended for systems that do not support it.
+
+easyblock = 'Tarball'
+
+name = 'CellRanger-ARC'
+version = '1.0.1'
+
+homepage = 'https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger'
+description = """Cell Ranger is a set of analysis pipelines that process Chromium
+ single-cell RNA-seq output to align reads, generate gene-cell matrices and perform
+ clustering and gene expression analysis."""
```

Comment:
```suggestion
description = """Cell Ranger ARC is a set of analysis pipelines that process
 Chromium Single Cell Multiome ATAC + Gene Expression sequencing data to generate a
 variety of analyses pertaining to gene expression, chromatin accessibility and
 their linkage. Furthermore, since the ATAC and gene expression measurements are on
 the very same cell, we are able to perform analyses that link chromatin
 accessibility and gene expression."""
```